convert-baichuan-hf-to-gguf.py

```diff
@@ -0,0 +1,293 @@
+#!/usr/bin/env python3
```

Comment:
This file has a shebang, so it should be made executable (mode 0755).

Reply:
got it